### ISSUE TITLE
Cannot Assign Project Manager Role to an individual feature

### ISSUE THREAD

**Issue by ghimiresushma99** [hotosm/tasking-manager#5241]
The feature to assign the project manager to a particular individual is not available. 

Whenever you try to do so, you need to provide a team name and assign a role to that team. You can neither select a person from that team to be a project manager. 

![image](https://user-images.githubusercontent.com/80664831/178650529-257a8ff5-98d4-431a-8367-bb99d9255e13.png)

**Comment by ghimiresushma99** [comment]
Yes, creating teams for each project makes sense, especially for urgent and disaster response tasks. But for a low-priority project, where only an individual's involvement as a project manager is sufficient, creating a team every time with just one individual sounds a little strange.

If 3 projects are being run and 3 different individuals are to be assigned the role of project manager, one should be making 3 teams for 3 people just to do a similar task, i.e. assign a project manager. What can be done instead is, to make a team where the individuals who qualify for the project managers will be team members and assign the project manager roles to them as per needed.  

This eliminates the repeated steps needed in forming many teams with a single member. Additionally, the organization will be managing fewer teams.

**Comment by petya-kangalova** [comment]
Wait to see what others think about the need. We should be aware that a lot of thought went into setting “teams” with the last Tasking Manager version. Before we go ahead with any changes, we need to really understand the challenge and how many users are affected. To put on hold for now until further discussion.
